### PR TITLE
Add test cases for ZookeeperConfiguration and ZookeeperProperties. #508

### PR DESCRIPTION
**Pull request by fengzhenbing**
Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ */
+
+package org.dromara.soul.admin.config;
+
+import org.I0Itec.zkclient.ZkClient;
+import org.apache.curator.test.TestingServer;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
```
Comment (songyuequan) [inline soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperConfigurationTest.java:25]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperPropertiesTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.admin.config;
+
+import org.junit.Test;
```
Comment (songyuequan) [inline soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperPropertiesTest.java:20]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperConfigurationTest.java (hunk 1 above)
Comment (fengzhenbing) [reply soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperConfigurationTest.java:25]: ok

File: soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperPropertiesTest.java (hunk 2 above)
Comment (fengzhenbing) [reply soul-admin/src/test/java/org/dromara/soul/admin/config/ZookeeperPropertiesTest.java:20]: ok

**Pull request closed without merging**